configs/mask2former/mask2former_r50_8xb2-lsj-50e_coco.py

```diff
@@ -56,6 +55,19 @@
     dict(type='PackDetInputs')
 ]
 
+test_pipeline = [
+    dict(
+        type='LoadImageFromFile',
+        file_client_args={{_base_.file_client_args}}),
+    dict(type='Resize', scale=(1333, 800), keep_ratio=True),
+    # If you don't have a gt annotation, delete the pipeline
```

Comment:
Delete. Already supported

Reply:
what's the meaning of Already supported?
the test pipeline in mask2former_r50_8xb2-lsj-50e_coco.py should like this?
test_pipeline = [
    dict(
        type='LoadImageFromFile',
        file_client_args=_base_.file_client_args),
    dict(type='LoadAnnotations', with_bbox=True, with_mask=True),
    dict(
        type='PackDetInputs',
        meta_keys=('img_id', 'img_path', 'ori_shape', 'img_shape',
                   'scale_factor'))
]

Reply:
I mean remove this comment